src/DIRAC/FrameworkSystem/Client/ProxyManagerClient.py

```diff
@@ -76,27 +76,22 @@ def __refreshUserCache(self, validSeconds=0):
         data = retVal["Value"]
         # Update the cache
         for record in data:
-            cacheKey = (record["DN"], record["group"])
+            cacheKey = record["DN"]
             self.__usersCache.add(cacheKey, self.__getSecondsLeftToExpiration(record["expirationtime"]), record)
         return S_OK()
 
     @gUsersSync
-    def userHasProxy(self, userDN, userGroup, validSeconds=0):
-        """Check if a user(DN-group) has a proxy in the proxy management
+    def userHasProxy(self, userDN, validSeconds=0):
+        """Check if a user(DN) has a proxy in the proxy management
         Updates internal cache if needed to minimize queries to the service
 
         :param str userDN: user DN
-        :param str userGroup: user group
         :param int validSeconds: proxy valid time in a seconds
 
         :return: S_OK()/S_ERROR()
         """
 
-        # For backward compatibility reasons with versions prior to v7r1
-        # we need to check for proxy with a group
-        # AND for groupless proxy even if not specified
-
-        cacheKeys = ((userDN, userGroup), (userDN, ""))
+        cacheKeys = ((userDN,), (userDN, ""))
```

Comment:
```suggestion
        cacheKeys = (userDN, )
```
I think you can drop the tuple with the empty group, since the only cacheKey that is created now is the userDN string(!) (line 79). So the only thing in here should be the userDN string, and not a tuple that only contains the userDN string! Right?

```
>>> d = dict()
>>> d['s'] = 123
>>> d[('s',)] = 321
>>> d
{'s': 123, ('s',): 321}
>>> for k in ('s', ('s',), ('s', "")):
...   print(d.get(k))
... 
123
321
None
```

Reply:
oh, yes. Done.